### DISCUSSION TITLE
퓨전 개발자 스터디 PL-900 회고(스터디 참여 후기)

### DISCUSSION THREAD

**Discussion by youngsunkr** [fusiondevkr/pl900#71] [1 Q&A]
좋은 스터디 주최해주신 두분 감사드립니다.

이제 월요일은 좀 쉬어볼까요 ^^;

https://blog.naver.com/youngsunkr/222493830366

**Comment by sesme100** [comment]
저 이 후기 읽고 가슴이 찡해졌어요! 덕분에 스터디 했길 잘 했구나 생각합니다. 다시한번 초짜의 마음 되새김질 합니다!! 후기 감사드립니다.
